### PR TITLE
feat(driving-license): allowing the quality photo provider to provide no image

### PR DESCRIPTION
**Pull request by arnorhs**
A few cleanups and fixes for the quality photo check

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/driving-license/src/dataProviders/QualityPhotoProvider.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class QualityPhotoProvider extends BasicDataProvider {
 
     return {
       success: response.data.qualityPhoto.success,
-      qualityPhoto: response.data.qualityPhoto.qualityPhoto,
+      qualityPhoto: response.data.qualityPhoto?.qualityPhoto,
```
Comment (saevarma) [inline libs/application/templates/driving-license/src/dataProviders/QualityPhotoProvider.ts:38]: Shouldn't the optional chaning operator also be used in L#37 and have default false on success if `qualityPhoto` can be `undefined` or `null`?

File: libs/application/templates/driving-license/src/fields/TeacherLookupField.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    },
+  })
+
+  if (queryError) {
+    return <Text>Villa kom upp við að sækja upplýsingar um nemanda</Text>
+  }
+
+  if (loading) {
+    return <Text>Sæki upplýsingar um nemanda... </Text>
+  }
```
Comment (saevarma) [inline libs/application/templates/driving-license/src/fields/TeacherLookupField.tsx:59]: Isn't the data about the teacher but not the student?

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 3]
```diff
@@ -311,6 +311,11 @@ export const m = defineMessages({
     defaultMessage: 'Ljósmynd í ökuskírteinaskrá',
     description: 'title for quality photo section',
   },
+  qualityPhotoAltText: {
+    id: 'dl.application.qualityPhotoAltText',
+    defaultMessage: 'Þín mynd skv ökuskírteinaskrá',
```
Comment (saevarma) [inline libs/application/templates/driving-license/src/lib/messages.ts:316]: Missing dot in "skv."

File: libs/application/templates/driving-license/src/lib/messages.ts [hunk 4]
```diff
@@ -356,14 +361,14 @@ export const m = defineMessages({
   qualityPhotoInstructionsBulletTwo: {
     id: 'dl.application:qualityPhoto.instructionsbullettwo',
     defaultMessage: `
-    Sýna þarf höfuð (án höfuðfats) og herðar 
+    Sýna þarf höfuð (án höfuðfats) og herðar
```
Comment (saevarma) [inline libs/application/templates/driving-license/src/lib/messages.ts:364]: Missing dot in the end of the sentance.

**Review by saevarma — CHANGES_REQUESTED** [review]
Minor comments

File: libs/application/templates/driving-license/src/fields/TeacherLookupField.tsx (hunk 2 above)
Comment (arnorhs) [reply libs/application/templates/driving-license/src/fields/TeacherLookupField.tsx:59]: actually.. my bad.. this wasn't supposed to be in there.. thank you!

**Review by saevarma — APPROVED** [review]
LGTM

**Review by alexandradilja — APPROVED** [review]

**Pull request merged**